docs-chef-io/content/inspec/resources/k8s_config_map.md

```diff
@@ -0,0 +1,74 @@
++++
+title = "k8s_config_map resource"
+draft = false
+gh_repo = "inspec"
+platform = "k8s"
+
+[menu]
+[menu.inspec]
+title = "k8s_config_map"
+identifier = "inspec/resources/k8s/K8s Config Map"
+parent = "inspec/resources/k8s"
++++
+
+Use the `k8s_config_map` Chef InSpec audit resource to test the configuration of specific Configuration Maps in a namespace.
+
+## Installation
+
+## Syntax
+
+```ruby
+describe k8s_config_maps(namespace: "NAMESPACE", name: "NAME") do
+  #...
+end
+```
+
+## Parameters
+
+`type`
+: Type of the K8s resource that is for a query.
```

Comment:
As we are setting up the type to `configmaps` internally I don't think `type` parameter is required here.